_maps/map_files/BirdStation/job_changes.dm

```diff
@@ -0,0 +1,138 @@
+#define JOB_MODIFICATION_MAP_NAME "BirdStation"
```

Comment:
This needs to be the same as `map_name` in the JSON iirc

Reply:
Oh, that's probably why it wasn't working. Makes sense.